### ISSUE TITLE
Your sample R project could use some beefing

### ISSUE THREAD

**Issue by deargle** [alexqaddourah/msbx5500-spring-2020#14]
You mentioned you applied to that one company that presented today. I saw that one of your code repos is R. Since that one guy seems to be an R guy, I'd recommend filling out that one project a bit. Looked like just a few lines of code? Needs some pizzaz.

**Comment by alexqaddourah** [comment]
Yep, I know and I am going through it all again. Adding better comments inside the code, making the code flow better and then going to do some nice markdown in GH explaining what I did and add some screenshots for any visualization pieces I did. Will be done today.

**Comment by alexqaddourah** [comment]
Is it good practice to put in markdown like ALL the cleaning? Or could I say something like "reference the code posted for cleaning" and then here's what I did after everything was clean...?

**Comment by deargle** [comment]
Meh, on one extreme, you can be a lazy cuss and assume that the code is self-explanatory. On the other, you can assume the reader would be completely lost without code comments. on that note, you can do a mix of code comments plus markdown. I might do something like:

# Data cleaning
we remove outliers, fill in missing values using medians grouped by blah-blah, and normalize the data.

```
data_no_outliers <- data.obscure_function # remove points > 2.5 std dev
```

**Comment by alexqaddourah** [comment]
Okay sounds good I'll do something like it

**Comment by deargle** [comment]
If you like. That's just my style,

**Comment by alexqaddourah** [comment]
How does it look now?

**Comment by deargle** [comment]
Link because I'm lazy?

**Comment by alexqaddourah** [comment]
https://github.com/alexqaddourah/Denver-AirBnb-Project

**Comment by deargle** [comment]
Holy headers batman! `#` in markdown converts to an html header tag. Inside of a codeblock, it's typically a "comment" depending on the language.

`## in markdown this will be header two, but not when it's inside of backticks`
## in markdown this will be header two, but not when it's inside of backticks

`#### in markdown this will be header four (<h4>), but not when it's inside of backticks`
#### in markdown this will be header four (`<h4>`), but not when it's inside of backticks

**Comment by deargle** [comment]
Otherwise, nice beefy content at first glance! Fix the headers and I'll try reading it again. Ay my eyes.

**Comment by alexqaddourah** [comment]
I wasn't intending for it to be comments though. Maybe I am misunderstanding what you're saying.

**Comment by deargle** [comment]
I doubt you were intending for text paragraphs to be formatted as headers -- why did you prefix all of them with `#` signs? Remove the pounds, and the paragraphs will become... paragraphs ( will convert to `<p>`)

**Comment by deargle** [comment]
![image](https://user-images.githubusercontent.com/1174653/73864909-a7764500-47ff-11ea-9c05-9bce24212689.png)

**Comment by alexqaddourah** [comment]
Got it got it lol I removed them all
